android-sdk/tasks/main.yml

```diff
@@ -75,15 +83,15 @@
   when: accept is undefined
 
 - pause:
-    prompt:  "{{ output.stdout }} Hit enter to accept the Licence or hit Ctrl _c and 'A' to reject and cancel the installation "
+    prompt:  "{{ rsh_cmd.stdout.split('--------------------')[1] }} Hit enter to accept the Licence or hit Ctrl _c and 'A' to reject and cancel the installation "
   when: accept is defined and accept
 
 - name: "Accepting the licence"
```

Comment:
@PhilipGough Could we also add something like "Please be patient. This may take a long time." or similar to this task as well. The androidctl sync can take some time.